### PR TITLE
2021.3.x feature split cred check

### PR DESCRIPTION
**Pull request by darh**
Please take a look.

### CODE REVIEW

File: auth/handlers/handle_login.go [hunk 1]
```diff
@@ -17,26 +19,66 @@ type (
...
+				// User w/o the password
+				// In case there is one single IdP automatically redirect user there
+				req.RedirectTo = fmt.Sprintf("%s/%s", GetLinks().External, h.Settings.Providers[0].Handle)
+
+				// for the clarity of the flow
+				// keeping this optional return here
+				return
+			}
+
+			// This user has existing password credentials or with more than one IdP
```
Comment (petergrlica) [inline auth/handlers/handle_login.go:69]: log debug that there is more than one idp?

**Review by petergrlica — APPROVED** [review]
looks good, just one comment/question

File: auth/handlers/handle_login.go (hunk 1 above)
Comment (darh) [reply auth/handlers/handle_login.go:69]: Fair point. tnx

**Comment by darh** [comment]
merged.

**Pull request closed without merging**